frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/initial-state/vm-settings-tab-initial-state.ts

```diff
@@ -44,6 +44,8 @@ export const getInitialVmSettings = (data: CommonData): VMSettings => {
     [VMSettingsField.DESCRIPTION]: {},
     [VMSettingsField.USER_TEMPLATE]: {
       isHidden: hiddenByProviderOrTemplate,
+      initialized: isCreateTemplate || !userTemplateName,
+      value: userTemplateName || undefined,
```

Comment:
you can set `isDisabled` with `asDisabled` function which has a disabled requestor id as a second argument. You can pass `VMWizardProps.userTemplateName` and it should not be overriden by anything else

Reply:
done :+1: